marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.queries.tsx

```diff
@@ -361,6 +361,12 @@ describe('save/manage queries scenarios, developer role', () => {
         browsePage.getExploreButton().click();
     })
 
+    it('verify export array/structured data warning', () => {
+        browsePage.selectEntity('Order');
+        browsePage.getSelectedEntity().should('contain', 'Order');
+        browsePage.getDataExportIcon().click();
+        browsePage.getStructuredDataWarning().should('be.visible');
```

Comment:
Must verify the warning message. This can be verified via RTL as well, just FYI.